corehq/apps/hqadmin/management/commands/force_web_user_password_reset.py

```diff
@@ -0,0 +1,90 @@
+import uuid
+
+from django.conf import settings
+from django.core.management import BaseCommand, CommandError
+from django.urls import reverse
+
+from corehq.apps.hqwebapp.tasks import send_html_email_async
+from corehq.apps.users.models import WebUser
+from corehq.util.argparse_types import utc_timestamp
+from dimagi.utils.web import get_url_base
+
+
+class Command(BaseCommand):
+    """
+    For each user in newline-separated `web_users_file`, log the user out and force them to reset their password.
+    It also sends them an email with some context and a link to where they can initiate the password reset.
+
+    Usage:
+        $ ./manage.py force_web_user_password_reset web_users_file
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('web_users_file')
+        parser.add_argument('--skip-if-reset-since', type=utc_timestamp,
+                            help="Timestamp of the last_password_set after which we will not force a reset."
+                                 " Format as YYYY-MM-DD HH:MM:SS")
+
+    def handle(self, web_users_file, skip_if_reset_since, **options):
+        web_user_usernames = get_lines_from_file(web_users_file)
+        errors = []
+        web_users_to_reset = []
+        web_users_to_skip_due_to_recent_reset = []
+        for web_user_username in web_user_usernames:
+            web_user = WebUser.get_by_username(web_user_username)
+            if web_user is None:
+                errors.append(f'No user named {web_user_username}')
+            elif not web_user.is_web_user():
+                errors.append(f'{web_user_username} is not a WebUser')
+            elif skip_if_reset_since and web_user.last_password_set \
+                    and web_user.last_password_set > skip_if_reset_since:
+                web_users_to_skip_due_to_recent_reset.append(web_user)
+            else:
+                web_users_to_reset.append(web_user)
+
+        if errors:
+            raise CommandError(f'The following errors were found in your input file:\n{errors}')
```

Comment:
Nit: including errors this way may not be the most readable (assuming the output gets put on the command line). It might make sense to join the errors array with newlines